pebblo/app/libs/logger.py

```diff
@@ -3,11 +3,17 @@
 """
 import logging
 
+LOG_LEVELS = {}
+
 
 def get_logger():
     """Get uvicorn logger"""
+    global LOG_LEVELS
```

Comment:
Instead of global variable, let's get logging levels in validate method directly. This variable wont be used anywhere else.

Reply:
Revised the code to reflect the following improvements.